unity-client/Assets/Scripts/MainScripts/Debugging/FPSDisplay/FPSEvaluation.cs

```diff
@@ -2,6 +2,8 @@
 {
     public class FPSEvaluation
     {
+        public const float HICCUP_THRESHOLD_IN_SECONDS = 0.05f;
```

Comment:
Nitpick, but the threshold for instant response we were checking out wasn't of 55ms?

Reply:
it was 33ms (1/30FPS)